src/main/java/core/basesyntax/DataLineInfo.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class DataLineInfo {
+    String date;
```

Comment:
what about access modifiers, should they be default?

Reply:
updated. done